storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/UpdateQuery.java

```diff
@@ -81,9 +79,34 @@ public int hashCode() {
          * @return builder
          */
         @NonNull
-        public Builder uri(@NonNull Uri uri) {
+        public CompleteBuilder uri(@NonNull Uri uri) {
             this.uri = uri;
-            return this;
+            return new CompleteBuilder(this);
+        }
+
+        /**
+         * Required: Specifies uri
+         *
+         * @param uri URI to query. This can potentially have a record ID if this is an update request for a specific record
+         * @return builder
+         */
+        @NonNull
+        public CompleteBuilder uri(@NonNull String uri) {
+            this.uri = Uri.parse(uri);
+            return new CompleteBuilder(this);
+        }
+    }
+
+    /**
+     * Compile-time safe part of {@link Builder}
+     */
+    public static class CompleteBuilder extends Builder {
+
+        private String where;
+        private List<String> whereArgs;
+
+        CompleteBuilder(@NonNull Builder builder) {
+            uri = builder.uri;
```

Comment:
If we do smth like this
.uri("usersUri").where("name = 'Ivan'").uri(anotherUsersUri);
Will we lost where clause?